ci-operator/step-registry/addon/uninstall/addon-uninstall-ref.yaml

```diff
@@ -9,16 +9,22 @@ ref:
   env:
     - name: ADDON1_CONFIG
       default: ""
-      documentation: Addon1 name
+      documentation: Addon1 name such as 'name=ocm-addon-test-operator'
     - name: ADDON2_CONFIG
       default: ""
-      documentation: Addon2 name
+      documentation: Addon2 name such as 'name=ocm-addon-test-operator'
     - name: ADDON3_CONFIG
       default: ""
-      documentation: Addon3 name
+      documentation: Addon3 name such as 'name=ocm-addon-test-operator'
     - name: ADDON4_CONFIG
       default: ""
-      documentation: Addon4 name
+      documentation: Addon4 name such as 'name=ocm-addon-test-operator'
+    - name: ADDON5_CONFIG
+      default: ""
+      documentation: Addon5 name such as 'name=ocm-addon-test-operator'
+    - name: ADDON6_CONFIG
+      default: ""
+      documentation: Addon6 name such as 'name=ocm-addon-test-operator'
     - name: PARALLEL
       default: ""
       documentation: To run addons uninstall in parallel, set to "true"
```

Comment:
Can be removed:
``` 
 - name: ROSA
    default: ""
    documentation: To uninstall addon via rosa cli, set to "true"
```

Reply:
This is not needed; rosa is part of the addon dict params